dns_managed_zone_with_wildcard_certificate/main.tf

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "domain" {
+  default     = "skiski.cloud"
+  description = "setup your domain"
+}
+
+variable "name" {
+  default     = "examplename"
+  description = "the name prefix for your resources"
+}
```

Comment:
Can these be locals instead?

Reply:
yes, this will be replace by locals in next commit, is there any other problem need to be solved?